content/docs/start/model-management/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Get Started: Model Management'
+description:
+  'Get started with DVC model management. Use the DVC model registry to manage
+  the lifecycle of your models in an auditable way. Easily access your models
+  and integrate your model registry actions into CICD pipelines to follow GitOps
+  best practices.'
+---
+
+# Get Started: Model Management
+
+## Chapters
+
+- **[Model registry]** - Set up a git-based model registry with DVC to track and
+  manage models, their versions and lifecycle stages.
+
+- **[Using and deploying models]** - Easily download your models from the model
+  registry. Set up your CICD pipelines to be trigger by model registry actions
+  (such as assigning model stages) and deploy models directly form the model
+  registry.
+
+[model registry]: /doc/start/model-management/model-registry
+[Using and deploying models]: /doc/start/model-management/model-deployment
+
+<admon type="tip">
+
+<!---
+TODO: Replace the links to repos with MR getting-started repo
```

Comment:
Do we need a new project/repo? Why can't we reuse https://github.com/iterative/example-get-started-experiments?

Reply:
(copying from Slack to keep the conversation here) Well, the tracks should probably be separate in the docs but I am wondering whether to keep then in the same repo with experiments. Separate repos means more flexibility but also more maintenance. And I think that the MR repo can rally just continue where the getting-started-experiments ends (with some slight reshuffling). On the other hand making a separate repo is probably easier at the moment and then we can compare them and see if they can be merged for ease of maintenance.